react-ui/src/app/dashboard/household/new-household.js

```diff
@@ -202,7 +232,7 @@ export default class NewHousehold extends React.Component<
                     </Alert>
                 )}
                 <HouseholdForm
-                    data={this.state}
+                    data={this.data}
```

Comment:
`this.data` or `this.state.data`?

Reply:
Nevermind-- tested and working

Reply:
Using a getter to get data off of state.   

```
get data() { this.state.data}
```